tests/test_tuners_utils.py

```diff
@@ -302,9 +304,9 @@ def test_maybe_include_all_linear_layers_internals(self, initial_target_modules,
         new_config = _maybe_include_all_linear_layers(config, model)
         if isinstance(expected_target_modules, list):
             # assert that expected and actual target_modules have the same items
-            self.assertCountEqual(new_config.target_modules, expected_target_modules)
+            assert set(new_config.target_modules) == set(expected_target_modules)
```

Comment:
Note that `assertCountEqual` is not equivalent to comparing sets. For this specific test, it doesn't matter, as there are no duplicates, so it's fine. Also, `new_config.target_modules` is already a set, so the `set` call is unnecessary.

Reply:
I know it's not – this was a change made by hand, looking at the code around it :)